test/test_saemref.py

```diff
@@ -84,3 +85,61 @@ def test_saemref_running(Process, Service, Socket, Command, supervisor_service_n
 def test_saemref_sync_source_cronjob(Command):
     jobs = Command.check_output("crontab -u saemref -l")
     assert 'cubicweb-ctl source-sync --loglevel error saemref' in jobs
+
+
+@wait_saemref_started
+def test_saemref_client(Process, Command, LocalCommand, TestinfraBackend):
+    docker_id = TestinfraBackend.hostname
+    # transfer local c-c shell file to the docker and execute it
+    copy_to_docker(LocalCommand, 'configure_instance.py', docker_id)
+    output = cc_shell(Command, 'configure_instance.py')
+    # retrieve the generated secret token
+    token_secret = output.splitlines()[-1]
+    # and generate file containing authentication information
+    output = Command.check_output(
+        '''cat <<EOF > cubicweb.yaml
+id: token-user
+secret: {}
+EOF'''.format(token_secret))
+
+    # test skos-download command
+    output = Command.check_output(
+        'python -m saemref_client skos-download -v http://localhost:8080 '
+        '25651/v1 -o /tmp/skos-download/')
+    folder_content = Command.check_output('ls /tmp/skos-download/25651-v1/*.xml')
+    assert folder_content, \
+        '{}\n\nFolder content: {}'.format(output, folder_content)
+
+    # test eac-upload command
+    copy_to_docker(LocalCommand, 'eac.xml', docker_id)  # transfer local EAC file to the docker
+
+    output = Command.check_output(
+        'python -m saemref_client eac-upload http://localhost:8080 '
+        '/tmp/eac.xml')
+    print('eac-upload', output)
+    assert 'ark' in output
+
+    # eac-download command
+    copy_to_docker(LocalCommand, 'publish_authority_records.py', docker_id)
+    print cc_shell(Command, 'publish_authority_records.py')
+    Command.check_output('mkdir /tmp/eac-download/')
+    output = Command.check_output(
+        'python -m saemref_client eac-download http://localhost:8080 '
+        ' -o /tmp/eac-download/')
+    folder_content = Command.check_output('ls /tmp/eac-download/*.xml')
+    assert folder_content, \
+        '{}\n\nFolder content: {}'.format(output, folder_content)
+
+
+def copy_to_docker(LocalCommand, fname, docker_id):
+    fpath = path.join(path.dirname(__file__), 'data', fname)
+    assert path.exists(fpath)
+    LocalCommand.check_output('cat %s | docker exec -i %s tee /tmp/%s',
+                              fpath, docker_id, fname)
```

Comment:
I think using a docker volume for test/data is better than copying files to the containter.

The feature already exists, see https://github.com/logilab/saemref-formula/blob/master/test/test_migration.py#L12